src/components/Settings/SettingsPlex.tsx

```diff
@@ -57,6 +55,10 @@ const messages = defineMessages({
   cancelscan: 'Cancel Scan',
   validationHostnameRequired: 'You must provide a valid hostname or IP address',
   validationPortRequired: 'You must provide a valid port number',
+  webAppUrl: 'Web App URL',
+  webAppUrlTip:
+    'Only required if not using the "hosted" <WebAppLink>web app</WebAppLink>',
```

Comment:
Not sure how I feel about this language. It's not required even if you don't use the hosted app (everything still works). And why is "hosted" in quotes?

Reply:
Ah, that's true about it not being required, I'll reword it. And I put "hosted" in quotes since Plex does in some places as well, but it probably doesn't need to be.